Cargo.toml

```diff
@@ -5,10 +5,11 @@ authors = ["The Cranelift Project Developers"]
 license = "Apache-2.0 WITH LLVM-exception"
 repository = "https://github.com/CraneStation/simplejit-demo"
 description = "Toy language implemented using cranelift-simplejit"
+edition = "2018"
 
 [[bin]]
 name = "toy"
-path = "src/toy.rs"
+path = "examples/toy.rs"
```

Comment:
It isn't an example, but a regular executable. It thus should be in `src/` not `examples/`.

Reply:
or maybe put in `src/bin/toy.rs`?